common/customizable_localization/00_divinity_custom_loc.txt

```diff
@@ -637,8 +637,83 @@ GetActualBishopTitle = {
 		}
 		localization_key = councillor_court_chaplain_vrykul_religion_county
 	}
+	# Jinyu
+	text = {
+		trigger = { 
+			liege.highest_held_title_tier >= tier_kingdom
+			faith = faith:waterspeaker
+		}
+		localization_key = councillor_court_chaplain_jinyu_religion_kingdom
+	}
+	text = {
+		trigger = { 
+			liege.highest_held_title_tier >= tier_duchy
+			faith = faith:waterspeaker
+		}
+		localization_key = councillor_court_chaplain_jinyu_religion_duchy
+	}
+	text = {
+		trigger = { 
+			liege.highest_held_title_tier < tier_duchy
+			faith = faith:waterspeaker
+		}
+		localization_key = councillor_court_chaplain_jinyu_religion_county
+	}
+	#Tirani
+	text = {
+		trigger = {
+			liege.highest_held_title_tier >= tier_kingdom
+			faith = faith:tidemother
+		}
+		localization_key = councillor_court_chaplain_tirani_religion_kingdom
+	}
+	text = {
+		trigger = {
+			liege.highest_held_title_tier >= tier_duchy
+			faith = faith:tidemother
+		}
+		localization_key = councillor_court_chaplain_tirani_religion_duchy
+	}
+	text = {
+		trigger = {
+			liege.highest_held_title_tier < tier_duchy
+			faith = faith:tidemother
+		}
+		localization_key = councillor_court_chaplain_tirani_religion_county
+	}
 
 	### Religions
+	# Secular
+	text = {
+		trigger = {
+			liege.highest_held_title_tier >= tier_kingdom
+			OR = {
+				faith = faith:mystery_of_the_makers
```

Comment:
Shenic faith still has priests.
If the Pantheonic faith gets reformed, chaplains become priests.
Thus I would make it default for the entire religion and not just one faith.

Reply:
done